server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/http/BasicAuthenticationTokenManagerTest.scala

```diff
@@ -0,0 +1,48 @@
+package org.apache.james.jmap.http
+
+import org.apache.james.core.Domain
+import org.apache.james.dnsservice.api.DNSService
+import org.apache.james.domainlist.memory.MemoryDomainList
+import org.apache.james.jmap.http.Fixture._
+import org.apache.james.user.api.UsersRepository
+import org.apache.james.user.memory.MemoryUsersRepository
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.{BeforeEach, Test}
+import org.mockito.Mockito.mock
+
+class BasicAuthenticationTokenManagerTest {
```

Comment:
We IMO need more tests here:

 - What if it is an empty string?
 - What if it is a non valid base 64 string?
 - `toBase64("")`
 - `toBase64("username")`
 - `toBase64("username:")`
 - `toBase64(":password")`
 - `toBase64("invalid@user@domain:password")`

ETC...

I would differenciate parsing from "authentication logic" in order to test it separately.